larky/src/main/java/com/verygood/security/larky/modules/VaultModule.java

```diff
@@ -0,0 +1,120 @@
+package com.verygood.security.larky.modules;
+
+import com.google.common.collect.Iterators;
+import com.verygood.security.larky.modules.vgs.vault.DefaultVault;
+import com.verygood.security.larky.modules.vgs.vault.spi.LarkyVault;
+import com.verygood.security.larky.modules.vgs.vault.NoopVault;
+import net.starlark.java.annot.Param;
+import net.starlark.java.annot.ParamType;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.NoneType;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.ServiceLoader;
+
+
+@StarlarkBuiltin(
+        name = "vault",
+        category = "BUILTIN",
+        doc = "Overridable Vault API in Larky")
+public class VaultModule implements LarkyVault {
+
+    public static final VaultModule INSTANCE = new VaultModule();
+    public static final String PROPERTY_NAME = "larky.modules.vgs.vault.enableInMemoryVault";
```

Comment:
Please, consider more descriptive var naming, `PROPERTY_NAME` is too generic imo.
Take it to the plain of the user of this library. Their code will look like:
```
env.set(VaultModule.PROPERTY_NAME, true)
```
The pearson who'll read it for the first time won't have a clue what is going on here.

Reply:
`VaultModule` would have been descriptive enough for this, if the property at question would enable the module and not enable specific mode of the module, like in this case. So more appropriate naming would be `ENABLE_INMEMORY_PROPERTY` for example.